mozdownload/scraper.py

```diff
@@ -762,7 +766,7 @@ def get_build_info(self):
             self.build_index = 0
             self.logger.info('Selected build: build%s' % self.build_number)
         else:
-            raise errors.NotSupportedError('Selected build not available')
+            raise errors.NotFoundError('Specified build has not been found', url)
```

Comment:
I assume that we can make this failure message specific to the build number like:

```suggestion
            raise errors.NotFoundError('Specified build number has not been found', url)
```